### DISCUSSION TITLE
Run multiple processes

### DISCUSSION THREAD

**Discussion by Kten987** [omkarcloud/google-maps-scraper#80] [Q&A]
The https://www.omkar.cloud/botasaurus/docs/google-maps-scraping-tutorial/ section is getting 503 Service Temporarily Unavailable so I can't read the instructions. In the technical usage section I see you mentioned Run multiple processes in parallel to minimize processing time. Where can I read it and where is its code?

**Comment by Chetan11-dev** [comment]
Learn Botasaurus here https://github.com/omkarcloud/botasaurus and the read scraping Code you will understand.
